elixir/apps/api/lib/api/gateway/channel.ex

```diff
@@ -83,13 +83,15 @@ defmodule API.Gateway.Channel do
       %{
         client_id: client_id,
         resource_id: resource_id,
+        flow_id: flow_id,
         authorization_expires_at: authorization_expires_at
       } = attrs
 
       resource = Resources.fetch_resource_by_id!(resource_id)
 
       push(socket, "allow_access", %{
         client_id: client_id,
+        flow_id: flow_id,
```

Comment:
@conectado this is the new ID that you will need to store in order to report metrics.